tests/unit/createRoute.ts

```diff
@@ -14,48 +14,6 @@ suite('createRoute', () => {
 		});
 	});
 
-	test('exec() can be customized', () => {
```

Comment:
I presume these are now removed due to being superfluous? as they would now just be testing compose behaviour?